### ISSUE TITLE
show a build shield for R-patched?

### ISSUE THREAD

**Issue by dtenenba** [metacran/r-builder#27]
Would it be possible to display a build shield for R-patched in addition to the ones displayed for R-devel at https://github.com/metacran/r-builder/#status-for-r-devel-builds ? I heard from someone that the build for R-patched was broken and there was no quick easy way to verify that.

Or am I not correct in assuming that the shields at the link above reflect the status of building R-devel, and not just packages under R-devel?
If not, I think it would be great if R itself (devel, release, patched, and aditional versions in the leadup to a release) could be built regularly and that shields could reflect the result of those builds.

**Comment by gaborcsardi** [comment]
Yes, those are for building R itself. Yes, I could have a badge for R-patched as well, although I guess that is tested, especially on Linux, so it should build. Some thoughts and issues:

- Often Travis has network problems, or the build timeouts, and then the badge is red. I don't usually do anything about this, the next build will be likely good.
- I don't build R-patched currently. I could, but need to set up the machinery for it first.
- I don't know how alpha, beta, rc, etc versions can be autodetected from the SVN. I guess they can, just don't know how.
- I don't run any tests for the R build currently. Maybe I should.

In summary, I agree that it would be nice to have a proper CI for R, maybe we could even have it in this repo. But maybe it is time to have an R-CI repo.

**Comment by gaborcsardi** [comment]
No need, no interest. Use the proper Travis support for R.
